CSharp/AccountManagement/Program.cs

```diff
@@ -1,334 +1,349 @@
+        // need to be a real endpoint, but must be a valid URI (e.g. https://accountmgmtsampleapp).
         private const string RedirectUri = "[specify-your-redirect-uri-here]";
-
-        // Specify the unique identifier (the "Client ID") for your application. This is required so that your
-        // native client application (i.e. this sample) can access the Microsoft Azure AD Graph API. For information
-        // about registering an application in Azure Active Directory, please see "Adding an Application" here:
-        // https://azure.microsoft.com/documentation/articles/active-directory-integrating-applications/
+
+        // Specify the unique identifier (the "Client ID") for your application. This is required so that your
+        // native client application (i.e. this sample) can access the Microsoft Azure AD Graph API. For information
+        // about registering an application in Azure Active Directory, please see "Adding an Application" here:
+        // https://azure.microsoft.com/documentation/articles/active-directory-integrating-applications/
         private const string ClientId = "[specify-your-client-id-here]";
-
-        // These constants are used by the ResourceManagementClient when querying AAD and for resource group creation.
-        // These values should not be modified.
-        private const string BatchNameSpace = "Microsoft.Batch";
-        private const string BatchAccountResourceType = "batchAccounts";
-
-        // The name of the Resource Group that will be created and deleted.
-        private const string ResourceGroupName = "AccountMgmtSampleGroup";
-
-        public static void Main(string[] args)
-        {
-            try
-            {
-                // Call the asynchronous version of the Main() method. This is done so that we can await various
-                // calls to async methods within the "Main" method of this console application.
-                MainAsync().Wait();
-            }
-            catch (AggregateException ae)
-            {
-                Console.WriteLine();
-                Console.WriteLine("One or more exceptions occurred.");
-                Console.WriteLine();
-
-                SampleHelpers.PrintAggregateException(ae.Flatten());
-            }
-            finally
-            {
-                Console.WriteLine();
-                Console.WriteLine("Sample complete, hit ENTER to exit...");
-                Console.ReadLine();
-            }
-        }
-
-        private static async Task MainAsync()
-        {
-            // Obtain an access token using the "common" AAD resource. This allows the application
-            // to query AAD for information that lies outside the application's tenant (such as for
-            // querying subscription information in your Azure account).
-            AuthenticationContext authContext = new AuthenticationContext(AuthorityUri);
-            AuthenticationResult authResult = authContext.AcquireToken(ResourceUri,

```

Comment:
💡 - Idea/Note - There is a formatting issue that's why the changes shows +364 -339.
For anyone wondering about the exact changes in this pull review: check this link which takes out whitespaces, https://github.com/Azure/azure-batch-samples/pull/135/files?w=   
Thanks,